active/AnimatedIcon/AnimatedIconSpec.md

```diff
@@ -280,6 +280,77 @@ XAML
 </ muxc:NavigationView.MenuItems >
 ```
 
+# Add an animated icon to a button 
```

Comment:
@MikeHillberg added example.

Reply:
Does this example provide value after the previous one? It's a lot of code and I think the only new thing it demonstrates is the FallBackSource feature. I think previously I'd asked for a button example, but that was when the spec said/implied that there would be built-in support for Button content.